docs/capture-visualiser/setting-up-the-rig.md

```diff
@@ -10,8 +10,8 @@ Setting up the Stage and Rigging
 To help you see what your rig will really look like you can add fixed
 elements of staging and rigging.
 
-Open the Capture Visualiser Settings window from the [Open Workspace
-Window menu](../titan-basics/workspace-windows.md#shortcuts-to-open-workspace-windows), or the \{Open Settings\} context button in the Capture window.
+Open the Capture Visualiser Settings window by double pressing \<View/Open\>,
```

Comment:
I appreciate it's confusing for D9 but a lot of current consoles do not have a View button. How do we refer to it so that all users understand what we are talking about?

Reply:
I do see what you mean. I suppose it's a result of the dots not quite being joined from the hardware to the software in the past.  It's almost a question of perspective - as far as the software is concerned, all of those consoles have View buttons (irrespective of what it says on the physical button / laminate), and the D9 is the first console to sport an Open button. I appreciate that this isn't clear to a user though.

I wonder if we should 
(a) have a specific section that explains this, and
(b) refer to the button as `View (legacy Open)` to disambiguate from the new `Open` button?

I'd be keen to find a way of clarifying it for both groups of users - as time goes on there will likely be new hardware throughout the range with separate View and Open, which means confusion restricted to the D9 currently will become more widespread.

@Farrser  - did we discuss having `\<Button Syntax\>` auto-generate links to the documentation for that button? If we did that, that'd be a great place for documentation that explains this.

Reply:
There is already a section which covers this in the Workspace Window section, though it could be made clearer.

I have done a poll of all consoles (from photos) and this is the result:
D9: View (and Open = View View)
ST: Open/View
Arena: Open/View
Quartz: Open
Titan Go: View
Titan Mobile: Open
Tiger Touch Mk2: Open/View
Tiger Touch Mk1: View
Pearl Expert: View

So the current "View/Open" is probably not helpful even for earlier consoles as mostly they have "Open/View".

As 99% of users are currently on an earlier console I would suggest we change the wording to "Open/View" throughout the manual and explain the D9 differences in the Workspace Window section. As more of the user base moves across to consoles with separate View and Open then we can update it.

Reply:
I think given where we're at, it's probably best for it to be "Open/View". On all the consoles, a double tap of the "Open/View" key will do the function of open, or you can just use "View" -> "Open workspace window".

Therefore I don't think it's too problematic if people get the view key confused with open for the time being. As we go forward, I think that button documentation is probably required as @p--b says. Ideally this would link the <View/Open> syntax to a page explaining where this button can be found on various consoles, and broadly what it's for.

As you say @congoblue most of our users are going to be on the previous generation of button layout at the moment. As the new button layout makes its way across more of the range I think we should look at migrating this across. If the function of the view button diverges more from the open button we might need to expedite that too.

For now, I think we can live with <Open/View> for my money and look at adding links to button specific pages later down the line.

Reply:
This sounds a good approach to me. 
I have added an issue #325 to create the button reference section.